pkg/fleet/telemetry/telemetry.go

```diff
@@ -95,6 +95,12 @@ func (t *Telemetry) Start(_ context.Context) error {
 		tracer.WithGlobalTag("site", t.site),
 		tracer.WithHTTPClient(t.client),
 		tracer.WithLogStartup(false),
+
+		// We don't need the value, we just need to enforce that it's not
+		// the default. If it is, then the tracer will try to use the socket
+		// if it exists -- and it always exists for newer agents.
+		// If the agent address is the socket, the tracer overrides WithHTTPClient to use it.
+		tracer.WithAgentAddr("localhost:8126"),
```

Comment:
this is a real trace-agent address, lets say that by mistake one day it gets used, a dummy one would be better: https://en.wikipedia.org/wiki/Reserved_IP_addresses ex 192.0.2.0:24?